test/unit/src/Test_common_includes.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TEST_COMMON_INCLUDES_H
+#define TEST_COMMON_INCLUDES_H
+
+
+// std includes
+#include <stdbool.h>
+
+
+// project c includes
```

Comment:
Not sure if this test_common_includes.h and test_includes.h is necessary? 
I would also try to stick to snake_case o kebak-case if possible for the files which are not required to start with capital case. Or files which do not include cpp classes, which usually use CamelCase.

Reply:
I fixed the test file name to snake case already, I remove one of the include file test_common_includes.h but kept test_includes.h to include unity header files... We can include only test_includes.h file in all the tests.